helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -65,3 +65,17 @@ spec:
         team: atlas
         topic: observability
         cancel_if_outside_working_hours: "true"
+    ## Page Atlas if prometheus agent fails to send samples to MC prometheus.
+    - alert: PrometheusAgentRemoteWriteNotWorking
+      annotations:
+        description: {{`Prometheus agent {{$labels.installation}}/{{$labels.cluster_id}} remote write is failing.`}}
+        summary: Prometheus agent fails to send samples to remote write endpoint.
+        opsrecipe: prometheus-agent-remote-write-failed/
+      expr: absent(sum(prometheus_remote_storage_samples_failed_total{instance="prometheus-agent"}) by (installation, cluster_id, instance, url)) > 0
```

Comment:
With `absent`, I guess you're looking for places where there's no `prometheus-agent` instances.
But:
* you're looking for `failed` counters. Are you sure they're created if all is OK? If there's no failed samples? You'd rather look for `prometheus_remote_storage_bytes_total` I think
* I don't understand the point of `sum` before `absent`.

Reply:
`prometheus_remote_storage_samples_failed_total`  is set to 0 if there's no errors!

![Screen Shot 2022-10-31 at 4 22 31 PM](https://user-images.githubusercontent.com/15221272/199044462-b2f3a849-a911-4641-9393-a854ce0b6817.png)